src/components/button-bar/button-bar.test.js

```diff
@@ -6,6 +6,10 @@ import {
 } from "./button-bar.config";
 
 import { buttonDataComponent } from "../../../cypress/locators/button";
+import ButtonBar from "../button-bar/button-bar.component";
```

Comment:
Can you add this as a new wrapped children story in to `button-bar-test.stories` please?